modules/Rock.cmake

```diff
@@ -56,6 +56,9 @@ macro (rock_init PROJECT_NAME PROJECT_VERSION)
     rock_use_full_rpath("${CMAKE_INSTALL_PREFIX}/lib")
     include(CheckCXXCompilerFlag)
     include(FindPkgConfig)
+    
+    OPTION(ROCK_USE_CXX11 "Compile package using the C++11 standard" ON)
```

Comment:
> like the widely used control/urdfdom version 1.0 from ROS.

It (and its dependencies) compile just fine on a Rock install that does not have C++11 enabled.

But anyways, this is *not* the point.

I am all for controlling that feature on the autoproj level, and turning it on by default on 16.04, optionally on older Ubuntus.

Now, base/cmake and orogen are two packages that are used outside of Rock, are used within Rock on older installations and older ubuntus. There is virtually no cost into keeping the current behavior and enabling it at the autoproj level for 16.04.